hs_core/views/__init__.py

```diff
@@ -168,7 +168,7 @@ def update_key_value_metadata(request, shortkey, *args, **kwargs):
         err_message = ex.message
 
     if is_update_success:
-        resource_modified(res, request.user)
+        resource_modified(res, request.user, overwrite_bag=False)
```

Comment:
Overwriting the bag, generates the xml metadata files. I want to avoid that so that metadata xml files are not generated each time any metadata element gets updated.